packages/manager/apps/web/client/app/hosting/cdn/order/translations/Messages_fr_FR.json

```diff
@@ -1,8 +1,30 @@
 {
-  "hosting_cdn_order_title": "Commander un CDN",
-  "hosting_cdn_order_description": "L'option CDN active un réseau de serveurs de cache pour vos fichiers statiques (HTML, CSS, JavaScript). Il vous permet d'accélérer le chargement de vos pages web et d'utiliser les ressources de votre hébergement uniquement pour vos pages dynamiques.",
+  "hosting_cdn_order_title": "Vous allez souscrire l’option Shared CDN",
+  "hosting_cdn_upgrade_title": "Migrer vers l’option Shared CDN",
+
+  "hosting_cdn_order_customer_cdn_included_step1_description": "Nous prenons en charge pour vous la migration de votre ancien CDN vers l’option Shared CDN. Nous migrerons également au fur et à mesure les noms de domaine de votre hébergement sur lesquels l’ancienne option CDN est activée. Une configuration par défaut sera mise en place. Les noms de domaine sur lesquels le CDN n’est pas activé resteront dans le même état après la migration.",
+  "hosting_cdn_order_customer_cdn_included_step1_header": "Prix de l’option",
+  "hosting_cdn_order_customer_cdn_included_step1_price": "La mise à jour vers le nouveau CDN est incluse avec votre solution Performance 1, 2, 3, 4 ou Cloud Web 1, 2 ou 3 sans facturation supplémentaire.",
+
+  "hosting_cdn_order_customer_cdn_payable_step1_description": "Nous prenons en charge pour vous la migration de votre ancien CDN vers l’option Shared CDN. Nous migrerons également au fur et à mesure les noms de domaine de votre hébergement sur lesquels l’ancienne option CDN est activée. Une configuration par défaut sera mise en place. Les noms de domaine sur lesquels le CDN n’est pas activé resteront dans le même état après la migration.",
```

Comment:
Same as `hosting_cdn_order_customer_cdn_included_step1_description` ?

Reply:
yes the both had same text but i created two key translations for each case